src/main/kotlin/solve/project/model/ProjectFrame.kt

```diff
@@ -2,4 +2,10 @@ package solve.project.model
 
 import java.nio.file.Path
 
-data class ProjectFrame(val timestamp: Long, val imagePath: Path, val landmarkFiles: List<LandmarkFile>)
\ No newline at end of file
+data class ProjectFrame(
+    val timestamp: Long,
+    val imagePath: Path,
+    val landmarkFiles: List<LandmarkFile>,
+    var errorMsg: MutableList<String> = mutableListOf(),
+    var errorType: MutableList<ErrorType> = mutableListOf()
```

Comment:
Create class structure of errors like it implemented with exceptions. Current implementation doesn't allow to match error messages with types

Reply:
Short names is prohibited. Use full names